src/cli_plugin/list/list.js

```diff
@@ -1,13 +1,28 @@
-import { statSync, readdirSync } from 'fs';
+import { statSync, readdirSync, readFileSync, accessSync, R_OK } from 'fs';
 import { join } from 'path';
 
+function readable(path) {
```

Comment:
> in this case it shouldn't be needed, readFileSync will already throw if the file isn't available.

True but I didn't want to put `logger.log(filename)` in the catch.

> For 5.0+, plugins are required to have package.json (the installer will fail without that file). You should be able to assume that they exist, and a thrown error would be appropriate if the file didn't exist or was not readable.

Ok, something like: `throw Error('Unable to read package.json for ' + filename)` ?